.gitignore

```diff
@@ -81,3 +81,6 @@ ntools/one-m/ansible/ansible.cfg
 # Just in time documentation
 docs/server/source/schema
 docs/server/source/drivers-clients/samples
+
+#ignore .idea
+.idea/
```

Comment:
What's `.idea/`? If it's something you want your local Git to ignore, but it's not something _everyone_ should ignore, then put it in `bigchaindb/.git/info/exclude`